### DISCUSSION TITLE
Timestamp properties in etw exporter

### DISCUSSION THREAD

**Discussion by treyhoffman84** [open-telemetry/opentelemetry-cpp#1182] [General]
Is there plans to include timestamps in the etw exporter properties? I see timestamps in the metadata related to the span but not in the event properties.

**Comment by lalitb** [comment]
I think this was not added purposefully, as event-time can be obtained from the timestamp at the ETW layer, thus saving on network traffic. Feel free to add optional support (compile-time macro?) if you think it's required.
